<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Document</class>
 <widget class="QWidget" name="Document">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Document</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDown" stretch="1,3">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutForTree">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="KiranColorBlock" name="widgetLeft" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonBackHome">
            <property name="styleSheet">
             <string notr="true">QPushButton { 
height: 30px; 
padding-left: 24px; 
padding-top: 5px; 
text-align: left 
}</string>
            </property>
            <property name="text">
             <string>Back Home</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TocTree" name="treeWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="indentation">
             <number>24</number>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutForTextBrowser">
       <item>
        <widget class="KiranColorBlock" name="widgetRight" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QTextBrowser{
background-color: transparent; 
padding: 15px 0 10px 5px
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="openLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KiranColorBlock</class>
   <extends>QWidget</extends>
   <header location="global">kiranwidgets-qt5/kiran-color-block.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TocTree</class>
   <extends>QTreeWidget</extends>
   <header>toc-tree/toc-tree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
